src/tests/validity/len.ab

```diff
@@ -0,0 +1,22 @@
+// Output
+// 5
+// 6
+// 3
+// 3
+// 6
+// 3
+
+let name = "Andrew"
+let fruits = ["apple", "banana", "cherry"]
+
+fun foo(ref text, ref arr) {
```

Comment:
Consider annotating the output, so we can see which usage failed.  Also consider reordering tests:
```
fun foo(ref text, ref arr) {
    echo "String reference: {len text}"
    echo "Array reference: {len arr}"
}

echo "String literal: {len "Hello"}"
echo "Array literal: {len [1, 2, 3]}"
echo "String variable: {len name}"
echo "Array variable: {len fruits}"
```

Reply:
Also, you're only testing the happy path. You might want to verify that it fails with other types, e.g. `len 42`, `len true` etc.

Reply:
@hdwalters We don't have tests for errors, do we? I'd consider this as a separate scope